speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -234,23 +205,25 @@ void* ResearcherMenuInit(void* p_nextMenu)
     void* param = GfuiMenuLoad("ResearcherMenu.xml");
     GfuiMenuCreateStaticControls(s_scrHandle, param);
 
-    // Task checkboxes controls
-    GfuiMenuCreateCheckboxControl(s_scrHandle, param, "CheckboxTaskLaneKeeping", NULL, SelectLaneKeeping);
-    GfuiMenuCreateCheckboxControl(s_scrHandle, param, "CheckboxTaskSpeedControl", NULL, SelectSpeedControl);
+    // Task radio button controls
+    int tasksId = GfuiMenuCreateRadioButtonListControl(s_scrHandle, param, "TaskRadioButtons", NULL, SelectTask);
```

Comment:
I would personally prefer to name the output `tasksControl`, because that's what we are creating. An `Id` can be almost anything, I feel like that naming is too generic.

NOTE: same goes for the other 2 radiobutton lists below.